engine/input/src/input.cpp

```diff
@@ -113,9 +113,13 @@ namespace dmInput
         }
     }
 
-    static GamepadConfig* GetGamepadConfigFromDeviceName(HBinding binding, const uint32_t device_hash)
+    static GamepadConfig* GetGamepadConfigFromDeviceName(HBinding binding, const uint32_t device_name_hash, const uint32_t generic_device_name_hash)
     {
-        GamepadConfig* config = binding->m_Context->m_GamepadMaps.Get(device_hash);
+        GamepadConfig* config = binding->m_Context->m_GamepadMaps.Get(device_name_hash);
+        if (config == 0x0)
```

Comment:
If no exact match is found we try with the generic device name instead

Reply:
An alternative is to ifdef on HTML5 here:
```
#if defined(__EMSCRIPTEN__)
if (!config) {
    config = binding->m_Context->m_GamepadMaps.Get(dmHashString("Standard Mapping"));
}
#endif
```

Reply:
Generic device id concept removed.